packages/core-data/src/actions.js

```diff
@@ -13,6 +14,11 @@ import { castArray } from 'lodash';
  * @return {Object} Action object.
  */
 export function receiveTerms( taxonomy, terms ) {
+	deprecated( 'wp.data.dispatch("core").receiveTerms', {
+		version: '13.6.0',
```

Comment:
Typo, it should be `3.6.0`. Actually, we should bump to `3.7.0` as we had new release in the meantime.